src/sentry/interfaces/http.py

```diff
@@ -173,7 +173,7 @@ def to_python(cls, data):
         body = data.get('data')
 
         try:
-            content_type = [h[1] for h in headers if h[0] == 'Content-Type'][0]
+            content_type = next((v for k, v in headers if k == 'Content-Type'), None)
```

Comment:
You can remove this from the `try/catch` block now (this will never throw.)

Reply:
5801a2b awesome yep